samples/update_protection_runs/README.md

```diff
@@ -0,0 +1,137 @@
+## Usage 
+```
+python --job_name "MSSQL-protect us-west-1" --start_time 2019-11-1 --end_time
+ 2019-11-18 --extend 10
+```
+
+## Connect to the Cohesity Cluster
+First make sure that you are connected to a Cohesity Cluster.
+```
+cohesity_client = CohesityClient(cluster_vip=CLUSTER_VIP,
+                                 username=CLUSTER_USERNAME, 
+                                 password=CLUSTER_PASSWORD,
+                                 domain=DOMAIN)
+```
+## Example
+``` 
+ body.job_runs.append(UpdateProtectionJobRun())
+ body.job_runs[index].job_uid = UniversalId()
+ body.job_runs[index].job_uid.cluster_id  = cluster_id
+ body.job_runs[index].job_uid.cluster_incarnation_id = incarnation_id
+ body.job_runs[index].job_uid.id = job_id
+ body.job_runs[index].run_start_time_usecs = run.copy_run[0].run_start_time_usecs
+ body.job_runs[index].copy_run_targets = []
+ body.job_runs[index].copy_run_targets.append(RunJobSnapshotTarget())
+ body.job_runs[index].copy_run_targets[0].days_to_keep = int(extend)
+ body.job_runs[index].copy_run_targets[0].mtype = \
+            TypeRunJobSnapshotTargetEnum.KLOCAL
+ index += 1
+ cohesity_client.protection_runs.update_protection_runs(body=body)
+
+```
+## Example Output
+```
+Start Time		                  Status		Success/Error		Extended Snapshot?
```

Comment:
Can we show previous values along with updated values?

Reply:
Good suggestion, but I think It's ok , consistent with our UI .